rules-tests/TypeDeclaration/Rector/ClassMethod/StrictStringParamConcatRector/Fixture/skip_concat_param_append_returned_used_as_arg.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictStringParamConcatRector\Fixture;
+
+class SkipConcatParamAppendReturnedUsedAsArg
+{
+    function formatSizeUnits($bytes): string
+    {
+        if ($bytes >= 1_073_741_824) {
+            $bytes = number_format($bytes / 1_073_741_824, 1).' GB';
```

Comment:
here `$bytes` need to be float, that's why it need to be skipped as possible non string type.